typescript/infra/scripts/sealevel-helpers/print-gas-oracles.ts

```diff
@@ -103,10 +103,13 @@ function getChainConnections(
       ['solanamainnet', 'everclear'],
       ['solanamainnet', 'infinityvmmainnet'],
       ['solanamainnet', 'sophon'],
-      // for svmBNB
+      // for svmBNB routes solana<>bsc<>svmbnb<>soon
```

Comment:
the comments above suggest these entries are unidirectional links (`Returns a record of origin chain -> set of chains that are connected via warp routes.`) rather than bidirectional. Which makes me wonder if we should also add `['solanamainnet', 'svmbnb'],`

Reply:
iiuc these are intended to be defined once and we'll add the bidirectional connectivity as part of a reduction afterwards https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/e82b98b583c0d6985ad850d021adf03059864847/typescript/infra/scripts/sealevel-helpers/print-gas-oracles.ts#L129-L147